### PR TITLE
update pom.xml

### PR DESCRIPTION
**Pull request by based2**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -27,19 +27,19 @@
...
+        <webdrivermanager.version>2.2.1</webdrivermanager.version>
         <phantomjsdriver.version>2.0.0</phantomjsdriver.version>
-        <slf4j.version>1.7.23</slf4j.version>
-        <testng.version>6.9.9</testng.version>
+        <slf4j.version>1.7.25</slf4j.version>
+        <testng.version>6.14.3</testng.version>
 
         <!-- Additional plugin versions -->
-        <maven.compiler.plugin.version>2.3.2</maven.compiler.plugin.version>
+        <maven.compiler.plugin.version>2.7.0</maven.compiler.plugin.version>
```
Comment (msaperst) [inline pom.xml]: this version doesn't appear to exist? Were you looking for 3.7.0?
https://mvnrepository.com/artifact/org.apache.maven.plugins/maven-compiler-plugin

Comment (based2) [reply pom.xml]: yes sorry

**Comment by msaperst** [comment]
It looks like a lot of the selenified integration tests aren't passing with this new versioning. My guess is it's the selenium bump. We'll need to determine why, and either update tests, or look for a version without bugs

**Comment by msaperst** [comment]
looks like this version bump is stable. I've been testing it for a while.
@grasbergerm @HammidFunsho any thoughts about merging it in?

**Pull request merged**